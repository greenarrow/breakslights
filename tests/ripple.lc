MA001
MR001
A000 C
A000 D249
A000 Fffffff
A000 L002
A000 IR
A000 S008
A000 Z0
MS000
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
