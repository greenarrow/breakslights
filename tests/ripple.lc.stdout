00 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000
00 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000
00 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000
00 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000
00 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000
00 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000
00 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff
00 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff
00 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff
00 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff
00 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff
00 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff
00 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff
00 ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff
00 ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff
00 ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff
00 ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff
00 ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff
00 ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff
00 ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff
00 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000
00 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000
00 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000
00 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000
00 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000
00 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000
00 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000
00 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff
00 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff
00 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff 000000 ffffff ffffff
